app/controllers/declaration/DestinationCountriesController.scala

```diff
@@ -51,25 +51,11 @@ class DestinationCountriesController @Inject()(
   implicit val countries = services.Countries.allCountries
 
   def displayForm(): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
-    handleForm(request.choice.value).map { resultView =>
-      Ok(resultView)
-    }
-  }
-
-  def saveCountries(): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
-    handleSaving(request.choice.value).flatMap {
-      case Left(resultView) => Future.successful(BadRequest(resultView))
-      case Right(call)      => Future.successful(Redirect(call))
-    }
-  }
-
-  private def handleForm(
-    decType: String
-  )(implicit request: JourneyRequest[AnyContent], hc: HeaderCarrier): Future[Html] =
-    decType match {
+    (request.choice.value match {
       case SupplementaryDec => handleSupplementary()
       case StandardDec      => handleStandard()
-    }
+    }).map(resultView => Ok(resultView))
+  }
 
   private def handleSupplementary()(implicit request: JourneyRequest[AnyContent], hc: HeaderCarrier): Future[Html] =
```

Comment:
Maybe `displayFormSupplementary` would be better? More clear about the purpose of it.